python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/dlt/asset_decorator.py

```diff
@@ -8,7 +8,7 @@
 from dlt.extract.source import DltSource
 from dlt.pipeline.pipeline import Pipeline
 
-from .constants import META_KEY_PIPELINE, META_KEY_SOURCE
+from .constants import META_KEY_PIPELINE, META_KEY_RESOURCE, META_KEY_SOURCE
```

Comment:
Thoughts on moving all metadata used for passing data from decorator -> runner into a single key?

Reply:
I think it's nice this way but if it gets out of hand we can change it